packages/ts-html-plugin/test/unsafe-tags.test.ts

```diff
@@ -11,7 +11,6 @@ it('Detect xss prone usage', async () => {
       <>
         <div>
         <div>{html}</div>
-        <div>{object}</div>
```

Comment:
@arthurfiorette I'm not sure about this, empty Objects are no longer supported by `kitajs/html`, right? Should we live this line by adding an expected ts error or should we, as I did, remove the line completely?

Reply:
Hmm, but the error thrown in this line is not from the ts-html-plugin and just a type error from typescript.

Reply:
Yep, that's correct. So we should not test this behavior. I'll leave it like this.